wgpu/src/lib.rs

```diff
@@ -1,6 +1,23 @@
-//! A cross-platform graphics and compute library based on [WebGPU](https://gpuweb.github.io/gpuweb/).
+//! `wgpu` is a cross-platform, safe, pure-rust graphics API. It runs natively on
+//! Vulkan, Metal, D3D12, and OpenGL; and on top of WebGL2 and WebGPU on wasm.
 //!
-//! To start using the API, create an [`Instance`].
+//! The API is based on the [WebGPU standard][webgpu]. It serves as the core of the
+//! WebGPU integration in Firefox, Servo, and Deno.
+//!
+//! [webgpu]: https://gpuweb.github.io/gpuweb/
+//!
+//! ## Getting Started
+//!
+//! The main entrypoint to the API is the [`Instance`] type, from which you can create [`Adapter`], [`Device`], and [`Surface`].
+//!
+//! If you are new to wgpu and graphics programming, we recommend starting by looking through
```

Comment:
style: backticks plz

style: moar breef moar happy

```suggestion
//! If you are new to `wgpu` and graphics programming, we recommend reading
```